config/jobs/kubernetes/sig-release/release-branch-jobs/1.20.yaml

```diff
@@ -1067,43 +1067,6 @@ presubmits:
         resources:
           requests:
             memory: 6Gi
-  - always_run: true
-    branches:
-    - release-1.20
-    cluster: k8s-infra-prow-build
-    labels:
-      preset-k8s-ssh: "true"
-      preset-service-account: "true"
-    max_concurrency: 12
-    name: pull-kubernetes-node-e2e
```

Comment:
Not sure it's good to remove it from 1.20 and 1.21 but if not config tests fail.